src/lightning_lite/cli.py

```diff
@@ -161,11 +174,10 @@ def _torchrun_launch(args: Namespace, script_args: List[str]) -> None:
     torchrun.main(torchrun_args)
 
 
-def main() -> None:
-    args, script_args = _parse_args()
+def main(args: Namespace, script_args: Optional[List[str]] = None) -> None:
     _set_env_variables(args)
-    _torchrun_launch(args, script_args)
+    _torchrun_launch(args, script_args or [])
 
 
 if __name__ == "__main__":
-    main()
+    _run_model()
```

Comment:
Since click is only used when this is called, this should check that click is available etc